client/components/userLineChart.jsx

```diff
@@ -0,0 +1,65 @@
+import { Line } from 'react-chartjs-2';
+import React from 'react';
+// eslint-disable-next-line no-unused-vars
+import Chart from 'chart.js/auto';
```

Comment:
I believe you can rewrite this to `import 'chart.js/auto';`

That way you can remove the ESLint directive.